app/assets/stylesheets/comments.scss

```diff
@@ -28,19 +41,124 @@
       border-left: 3px solid $blue;
       padding-left: 3px;
```

Comment:
- Nesting should be no greater than 3, but was 4
- Selector should have depth of applicability no greater than 3, but was 5